src/main/java/api/IBuilderApiResponse.java

```diff
@@ -0,0 +1,15 @@
+package api;
+
+public interface IBuilderApiResponse {
```

Comment:
Will we have another class that implement this interface? if not I think we don't need it